README.md

```diff
@@ -52,27 +52,41 @@ Conductor's configuration can be defined in both YAML and JSON formats. The conf
 
 ### Configuration File Example
 
-```yaml
-server:
-  port: 9000
-
-logger:
-  filter: error
-
-sources:
-  - type: graphql
-    id: my-source
-    config:
-      endpoint: https://my-source.com/graphql
-
-endpoints:
-  - path: /graphql
-    from: my-source
-    plugins:
-      - type: cors
-        config:
-          allowed_origin: "*"
-      - type: graphiql
```

Comment:
how about, we have to sections like this? 👀

### Configuration File Example (YAML)


```yaml
server:
  port: 9000

logger:
  filter: error

sources:
  - type: graphql
    id: my-source
    config:
      endpoint: https://my-source.com/graphql

endpoints:
  - path: /graphql
    from: my-source
    plugins:
      - type: cors
        config:
          allowed_origin: "*"
      - type: graphiql
```


### Configuration File Example (JSON)

```json
{
  "server": {
    "port": 9000
  },
  "logger": {
    "filter": "error"
  },
  "sources": [
    {
      "type": "graphql",
      "id": "my-source",
      "config": {
        "endpoint": "https://my-source.com/graphql"
      }
    }
  ],
  "endpoints": [
    {
      "path": "/graphql",
      "from": "my-source",
      "plugins": [
        {
          "type": "cors",
          "config": {
            "allowed_origin": "*"
          }
        },
        {
          "type": "graphiql"
        }
      ]
    }
  ]
}
```

Reply:
did